ts/octorelay.ts

```diff
@@ -207,11 +263,15 @@ $(() => {
           .on("click", () => toggleRelay(key, value));
         clearHints(relayBtn);
         if (hasUpcomingTask(value)) {
-          addPopover({ relayBtn, key, value, navbar });
+          upcomingTasks.push({ relayBtn, key, value });
         } else {
           addTooltip(relayBtn, value.label_text);
         }
```

Comment:
this should be organised in a different way: only the keys missing in `upcomingTasks` should get tooltips